app/assets/stylesheets/layout_components/buttons/button-grouping.scss

```diff
@@ -45,7 +45,19 @@
       background-color: $brand-green;
 
       &:hover{
-        background: lighten($brand-green, 10%);
+        background: lighten($brand-green, 10%)!important;
```

Comment:
why does this need !important? Also space before the !important

Reply:
all the hovers on our button groups were being over written by the hover set on .button-grouping__button. None of them had been working.

Reply:
hmm while this fixes the problem, I think we should try to fix the root problem instead. it looks like the issue is that we're extending .button-grouping__button. maybe take the extend out? you might have to duplicate some of the styles being added by that extend into the --green class.